lib/id003/id003.js

```diff
@@ -240,11 +238,25 @@ Id003.prototype._denominations = function _denominations () {
 }
 
 Id003.prototype._poll = function _poll () {
-  this._send('status')
+  this.rs232.send('status')
 }
 
 Id003.prototype._send = function _send (command, data) {
-  this.rs232.send(command, data)
+  this._stopPolling()
+  setTimeout(() => {
```

Comment:
Is this timeout necessary, since _startPolling uses and interval anyway and won't start immediately?

Reply:
Ahh, good point!